pkg/apis/kueueoperator/v1/types.go

```diff
@@ -70,6 +70,12 @@ type KueueConfiguration struct {
 	// This default could change over time.
 	// +optional
 	Preemption Preemption `json:"preemption"`
+	// multiKueue controls the behaviour of the MultiKueue AdmissionCheck Controller.
```

Comment:
So I was debating about two approaches.

1) Add API support for this for MK.
2) Automatically set up MK settings based on external framework fields.

Curious on your thoughts here?

Is there any reason why you want to configure external frameworks for tekton-kueue for single cluster but not have it enabled in case you are working in MK environment?

Reply:
I think the API support is fine.

It matches upstream and we reuse the same type so I don't think the API is too much to add.